### PR TITLE
Implement more reliable flow for biometric

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The original idea is borrowed from https://github.com/wix/AppleSimulatorUtils/blob/master/applesimutils/applesimutils/main.m

### CODE REVIEW

File: lib/simulator-xcode-9.js [hunk 1]
```diff
@@ -291,55 +282,61 @@ class SimulatorXcode9 extends SimulatorXcode8 {
...
+   * @override
    */
-  getBiometric (biometricName) {
-    const {BIOMETRICS} = SimulatorXcode9;
-    if (!BIOMETRICS[biometricName]) {
-      throw new Error(`${biometricName} is not a valid biometric. Use one of: ${JSON.stringify(_.keys(BIOMETRICS))}`);
+  async isBiometricEnrolled () {
+    const {stdout} = await spawn(this.udid, [
+      'notifyutil',
+      '-g', ENROLLMENT_NOTIFICATION_RECEIVER
```
Comment (KazuCocoa) [inline lib/simulator-xcode-9.js]: [nits]
What about adding `-v` to show a log with timestamp, key, etc?

File: lib/simulator-xcode-8.js [hunk 2]
```diff
@@ -210,13 +210,61 @@ class SimulatorXcode8 extends SimulatorXcode7 {
...
+
+  /**
+   * @inheritdoc
+   * @override
+   */
+  async isBiometricEnrolled () {
+    const output = await this.executeUIClientScript(`
+      tell application "System Events"
+        tell process "Simulator"
+          set dstMenuItem to menu item "Toggle Enrolled State" of menu 1 of menu item "Touch ID" of menu 1 of menu bar item "Hardware" of menu bar 1
```
Comment (dpgraham) [inline lib/simulator-xcode-8.js]: ~~Are we able to do the same thing for `Face Id`?~~

Comment (dpgraham) [reply lib/simulator-xcode-8.js]: Nvm, just remembered that `face id` doesn't apply to this xcode version.

File: lib/simulator-xcode-9.js [hunk 3]
```diff
@@ -291,55 +282,61 @@ class SimulatorXcode9 extends SimulatorXcode8 {
...
-        end tell
-      end tell
-    `);
+  async sendBiometricMatch (shouldMatch = true, biometricName = 'touchId') {
+    const domainComponent = toBiometricDomainComponent(biometricName);
+    const domain = `com.apple.BiometricKit_Sim.${domainComponent}.${shouldMatch ? '' : 'no'}match`;
+    await spawn(this.udid, [
+      'notifyutil',
+      '-p', domain
+    ]);
```
Comment (dpgraham) [inline lib/simulator-xcode-9.js]: Good find. This makes me feel a lot better than using AppleScript.

**Pull request merged**